daily_word_bot/db.py

```diff
@@ -62,6 +67,27 @@ def remove_user_blocked_word(self, message, word_id):
     def get_user_blocked_words(self, chat_id) -> typing.List[str]:
         return to_string_list(self.r.smembers(f"blockedWords-{chat_id}"))
 
+    def get_user_levels(self, chat_id) -> typing.List[str]:
+        user_info = self.get_user(chat_id)
+        user_levels = user_info.get("levels", [])
+        return user_levels
+
+    def remove_user_level(self, chat_id, level) -> None:
+        user_info = self.get_user(chat_id)
+        levels = user_info.get("levels", [])
```

Comment:
Estas líneas te las puedes ahorrar con el método de arriba get user levels. It applies as well for the method below :)

Reply:
pero necesito la info de user_info y la funcion solo devuelve los levels del usuario, como mucho podria cambiar user_info.get("levels", []) por get_user_levels pero aun y asi es como si la linea get_user se repitiera